src/routes/news/NewsPage.tsx

```diff
@@ -0,0 +1,106 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import LoadingProgress from "../../shared/components/LoadingProgress";
+import Select from "../../shared/components/Select";
+import { H2, screenMdMin } from "../../shared/styles";
+import NewsCard, { NewsArticle } from "./components/NewsCard";
+
+const NEWS_SELECT_OPTIONS = [
+  {
+    name: "Nutrition",
+  },
+  {
+    name: "Pharmaceuticals",
+  },
+  {
+    name: "Health",
+  },
+];
+
+const PageWrapper = styled.div`
+  padding: var(--spacing-md);
+`;
+
+const ArticlesWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+const HeaderWrapper = styled.div`
+  display: flex;
+  margin-bottom: var(--spacing-md);
+  justify-content: space-between;
+  @media ${screenMdMin} {
+    width: 65%;
+  }
+`;
+
+const StyledHeader = styled(H2)`
+  margin-left: var(--spacing-md);
+  text-decoration: underline;
+  @media ${screenMdMin} {
+    margin-top: var(--spacing-sm);
+  }
+`;
+
+const SelectWrapper = styled.div`
+  width: 200px;
+`;
+
+const NewsPage = () => {
+  const [newsArticles, setNewsArticles] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [articleType, setArticleType] = useState("Nutrition");
+
+  const getNewsArticles = async () => {
+    setLoading(true);
+    const articles = await axios.get(
+      `https://api.nytimes.com/svc/search/v2/articlesearch.json?q=${articleType}&api-key=${process.env.REACT_APP_NEWS_API_KEY}`
```

Comment:
I would extract the NY times url out to a constant. Maybe put it in a config file? But I don't think we have one yet so maybe not worth it. ENV would be overkill though I think